<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ThinDialog</class>
 <widget class="QDialog" name="ThinDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>819</width>
    <height>241</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="grp_pre_process">
     <property name="title">
      <string>Pre-process</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="lyt_shape" stretch="0,0">
        <item>
         <widget class="QLabel" name="lbl_shape_label">
          <property name="text">
           <string>Morphological Shape:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cmb_shape_value"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="lyt_k_size" stretch="0,0">
        <item>
         <widget class="QLabel" name="lbl_k_size_label">
          <property name="text">
           <string>K-Size (M, M):</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spb_k_size_value">
          <property name="minimumSize">
           <size>
            <width>75</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>0</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="lyt_pre_threshold_thresh" stretch="0,0">
        <item>
         <widget class="QLabel" name="lbl_pre_threshold_thresh_label">
          <property name="text">
           <string>Threshold - thresh:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="spb_pre_threshold_thresh_value">
          <property name="minimumSize">
           <size>
            <width>75</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>0</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="maximum">
           <double>100000.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="lyt_pre_threshold_max_value" stretch="0,0">
        <item>
         <widget class="QLabel" name="lbl_pre_threshold_max_value_label">
          <property name="text">
           <string>Threshold - maxValue:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="spb_pre_threshold_max_value_value">
          <property name="minimumSize">
           <size>
            <width>75</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>0</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="maximum">
           <double>100000.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="grp_in_process">
     <property name="title">
      <string>In process</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="lyt_max_intents" stretch="0,0">
        <item>
         <widget class="QLabel" name="lbl_max_intents_label">
          <property name="text">
           <string>Max intents:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spb_max_intents_value">
          <property name="minimumSize">
           <size>
            <width>75</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>0</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="minimum">
           <number>100</number>
          </property>
          <property name="maximum">
           <number>100000</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="lyt_branch_points" stretch="0,0">
        <item>
         <widget class="QLabel" name="lbl_branch_points_label">
          <property name="text">
           <string>Detect Branch-Points:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="chb_branch_points_value">
          <property name="text">
           <string>Yes</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="lyt_branch_points_color" stretch="0,0">
        <property name="rightMargin">
         <number>10</number>
        </property>
        <item>
         <widget class="QLabel" name="lbl_branch_points_color_label">
          <property name="text">
           <string>Branch-Points color:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="ColorLabel" name="lbl_branch_points_color_value">
          <property name="minimumSize">
           <size>
            <width>15</width>
            <height>15</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>15</width>
            <height>15</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="grp_post_process">
     <property name="title">
      <string>Post-process</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="lyt_post_threshold_thresh" stretch="0,0">
        <item>
         <widget class="QLabel" name="lbl_post_threshold_thresh_label">
          <property name="text">
           <string>Threshold - thresh:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="spb_post_threshold_thresh_value">
          <property name="minimumSize">
           <size>
            <width>75</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>0</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="maximum">
           <double>100000.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="lyt_post_threshold_max_value" stretch="0,0">
        <item>
         <widget class="QLabel" name="lbl_post_threshold_max_value_label">
          <property name="text">
           <string>Threshold - maxValue:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="spb_post_threshold_max_value_value">
          <property name="minimumSize">
           <size>
            <width>75</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>0</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="maximum">
           <double>100000.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorLabel</class>
   <extends>QLabel</extends>
   <header>subclasses/color_label.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
